src/components/RegisterForm/RegisterForm.tsx

```diff
@@ -33,20 +41,25 @@ export function RegisterForm() {
 
   const minLength = 1;
   const { t } = useTranslation();
-  const validate = (values: CreateUserType) => {
-    const errors: CreateUserType = {};
+  const validate = (values: CreateUserError) => {
+    let errors = {} as CreateUserError;
+
     if (values.email === undefined || values.email.length < minLength) {
-      errors.email = t('registerForm.emailEmpty');
+      errors = { ...errors, email: t('registerForm.emailEmpty') };
```

Comment:
why not just `errors.email = t('registerForm.emailEmpty')`. There is no need to create a copy of the whole object
(same comment to all below examples)